src/main/java/org/apache/commons/io/IOUtils.java

```diff
@@ -882,15 +894,26 @@ public static boolean contentEqualsIgnoreEOL(final Reader input1, final Reader i
                     nowPos2 += nowRead2;
                 }
             }
+
```

Comment:
To prevent accidental run-away, it would be safer to use >= to compare incremented indexes

Reply:
@sebbASF 
well it shouldn't in anyway...
If it really happened,I think there must bugs in jvm,and I don't think using >= can make things back to correct...

Reply:
If the index starts at a value below the limit, and you increment it by 1 between each check, then I agree that the values must be equal at some point, assuming single-threaded code.

However if there is a bug in the code that can cause the index to be incremented twice, it may never match.
Comparison using >= avoids this potential error.